finagle-http/src/main/scala/com/twitter/finagle/http/codec/ConnectionManager.scala

```diff
@@ -11,7 +11,7 @@ import com.twitter.util.{Future, Promise}
 private[finagle] class ConnectionManager {
   private[this] var isKeepAlive = false
   private[this] var isIdle = true
-  private[this] var activeStreams = 0
+  private[this] var activeStreams, pendingResponses = 0
```

Comment:
since ConnectionManager can be used for both clients and servers, this might be better phrased as `pending` and we should add a note that for servers it goes negative instead of positive.